app/scripts/lib/channels/web.js

```diff
@@ -46,11 +46,6 @@ define([
         sender: sender,
         receiver: receiver
       });
-    },
-
-    teardown: function () {
-      this._sender.teardown();
-      this._receiver.teardown();
```

Comment:
Indeed it is. The DuplexChannel [already takes care of this](https://github.com/mozilla/fxa-content-server/pull/2629/files#diff-cf353daf363493211804f545297ba732R88) so I piled on the unrelated change.